metrics/edfi_tech_metrics/ossf_score.py

```diff
@@ -1,10 +1,8 @@
-# SPDX-License-Identifier: Apache-2.0
-# Licensed to the Ed-Fi Alliance under one or more agreements.
-# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
-# See the LICENSE and NOTICES files in the project root for more information.
-
```

Comment:
Please restore this license header, which should be present on all of our open source code.